src/navigation/deep-linker.ts

```diff
@@ -70,7 +56,7 @@ import { ViewController } from './view-controller';
  * ```
  *
  * This Feels pretty familiar to how Angular sets up routes, but has some fundamental differences.
```

Comment:
There's not enough information in the rest of the doc for this sentence to be proven so let's just take it out